corehq/apps/reports/tasks.py

```diff
@@ -339,25 +353,41 @@ def _expose_download(fpath, use_transfer, zip_name, download_id, num_forms):
     DownloadBase.set_progress(build_form_multimedia_zip, num_forms, num_forms)
 
 
-def _write_attachments_to_file(fpath, use_transfer, num_forms, forms_info):
+def _format_filename(form_info, question_id, extension, case_id_to_name):
+    filename = u"{}-user_{}-form_{}{}".format(
```

Comment:
`user_` and `form_` were my addition cause i thought it was confusing to see two uuids in row